### PR TITLE
TENT-1489 - add thumbnails to email cards

### PR DESCRIPTION
**Pull request by rossedfort**
## Merge Checklist

Adds the ability for an email card to display a thumbnail.

[TENT-1489](https://jira.sendgrid.net/browse/TENT-1489)

- [x] Includes link to associated Github Issue or JIRA ticket
- [x] Has descriptive PR title and body
- [x] All changes are backwards compatible
- [x] QA'd by the SendGrid Style Guide designer
- [ ] Code reviewed by a SendGrid frontend lead

### CODE REVIEW

File: src/email-card.tsx [hunk 1]
```diff
@@ -101,21 +104,40 @@ const EmailCardContent: React.SFC<EmailCardContentProps> = ({
     )}
     {...attributes}
   >
-    <Button type="secondary" onClick={onContentEditClick}>
-      {hasContent ? (
+    {hasContent ? (
+      <div
```
Comment (cfree) [inline src/email-card.tsx]: Since this isn't semantic, can we add aria attributes?

File: src/flexHeader.tsx [hunk 2]
```diff
@@ -91,7 +91,7 @@ export class FlexHeader extends Component<FlexHeaderProps> {
         </div>
         <div className={cn(Styles['flex-header-right'], 'flex-header-right')}>
           {typeof creditsCount !== 'undefined' ? (
-            <Counter text="Credits" count={creditsCount} />
+            <Counter text="Available Credits" count={creditsCount} />
```
Comment (cfree) [inline src/flexHeader.tsx]: Travis thanks you

File: src/template-thumbnail.tsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    }
+  };
+
+  if (retryLimit && retryInterval) {
+    handleError = onImageLoadError;
+  } else {
+    handleError = noop;
+  }
+
+  return !failed && !!src ? (
```
Comment (twilio-FanLu) [inline src/template-thumbnail.tsx]: By just looking at `failed` and `src`, it is hard to know the what this means at a glance. Maybe `!reachRetryLimit && !!thumbnailUrl` or if other names that's more descriptive?

File: src/template-thumbnail.tsx [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+    handleError = onImageLoadError;
+  } else {
+    handleError = noop;
+  }
+
+  const showBlankTemplateImage = hasError || retryLimitReached || !thumbnailUrl;
+
+  return (
+    <>
+      <img
```
Comment (cfree) [inline src/template-thumbnail.tsx]: Since we aren't doing `srcset` attributes, we should minimally scale down the image for retina screens

Comment (rossedfort) [reply src/template-thumbnail.tsx]: I'm fairly certain we want to keep our thumbnails a fixed resolution and aspect ratio, this is something we would probably want to run by design. But in general, yes, we could certainly improve our app's responsiveness 👍

**Comment by thinkingserious** [comment]
Hello @rossedfort,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**